Docs/Configuration.tex

```diff
@@ -1751,9 +1734,9 @@ \subsection{Introduction}\label{devpropsintro}
 The second stage applies to all devices much later after the PCI configuration
 and may repeat the first stage if the device was not present in ACPI.
 
-For all kernel drivers, which may inspect the \texttt{IODeviceTree} plane without
-probing (e.g. \texttt{Lilu} and its plugins such as \texttt{WhateverGreen}) it is particularly
-important to ensure device presence in the ACPI tables. Failing to do so may result
+For all kernel drivers that may inspect the \texttt{IODeviceTree} plane without
+probing, such as \texttt{Lilu} and its plugins such as \texttt{WhateverGreen}, it is particularly
+important to ensure device presence in the ACPI tables. A failure to do so may result
```

Comment:
Suggest lighter changes to original would be preferable here:

```
For all kernel drivers which may inspect the \texttt{IODeviceTree} plane without
probing (e.g. \texttt{Lilu} and its plugins such as \texttt{WhateverGreen}) it is particularly
important to ensure device presence in the ACPI tables. Failure to do so may result
```

Reply:
See where you are coming from but that seems to be more from a preference perspective.
Made some changes but more in line with the original edit.

Reply:
cc @vit9696